contrib_bots/bots/yoda_bot/yoda_bot.py

```diff
@@ -17,13 +17,7 @@
 import logging
 import ssl
 import sys
-
-try:
-    import unirest
-except ImportError:
-    logging.error("Dependency missing!!\n%s" % (__doc__))
-    sys.exit(0)
-
+import requests
```

Comment:
I guess a error message is still needed since the bots isn't ran on the zulip server, and requests is a third party library.